src/settings/MappingProfiles/ViewMappingProfile.js

```diff
@@ -248,17 +248,17 @@ export class ViewMappingProfile extends Component {
     const marcMappingOptionLabel = FIELD_MAPPINGS_FOR_MARC_OPTIONS.find(option => option.value === marcMappingOption)?.label;
 
     const MARCBibDetailsProps = {
-      marcMappingDetails: mappingDetails?.marcMappingDetails,
+      marcMappingDetails: mappingDetails?.marcMappingDetails || [],
```

Comment:
What if create new const for result of 'mappingDetails?.marcMappingDetails || []'. Does it make sense?